blog/2022/11-18-tagging-as-a-treat/index.md

```diff
@@ -0,0 +1,143 @@
+---
+authors: [anja]
+tags: [aws, tagging, cloud]
+---
+
+# Tagging as a Treat
+
+Cloud Tagging Strategies and Policies are hailed as one of the most efficient ways to keep your cloud infrastructure controllable. But are they really?
+
+The idea generally is that every (taggable) piece of cloud service gets tagged (or _labelled_ in case of <abbr title="Google Cloud Platform">GCP</abbr>) by the developers or maintainers who work with it. This could be done with Infrastructure-as-Code Tools (see [Terraform](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/resource-tagging) for example), with a Command Line Interface or manually in the cloud UI.
+
+Tagging policies could state that every resource needs a tag to identify the owner, the cost center, the product, the project or any other metadata. By doing this diligently every resource can be managed via its tags and nothing gets overlooked.
+
+While in theory this is the right way to do it, in practice this hardly ever works as intended. Every tag created is a tag that needs maintaining. Tagging policies may change over time and people can make honest mistakes when tagging (in <abbr title="Amazon Web Services">AWS</abbr> for example: tag keys are case sensitive, which can quickly go wrong). To properly use tagging on a greenfield cloud account is one thing - to retroactively apply tags to sprawling cloud infrastructure is quite another (especially when utilising a multi-cloud strategy, where you’d need to repeat any operation over multiple interfaces).
+
+We [asked our community](https://www.linkedin.com/feed/update/urn:li:activity:6987739499686428672) about their most challenging aspect of cloud tagging strategies:
+
+![LinkedIn Poll](./img/linkedinpoll.png)
+
+Over 70% agree that human error is the biggest issue. Be it on the console, in the UI, in templates or in Terraform config files, tags are initially being typed by people. But then we want them to be properly understood by machines.
+
+As highlighted in [this recent blog post](../10-14-a-tale-of-two-tools/index.md) going through cloud infrastructure with CLI can be tiresome and requires a lot of contextual knowledge, as the API can differ between services. This is hardly conducive to effectively keeping tags under control. In that post we also outlined how Resoto simplifies interaction with your cloud(s).
```

Comment:
```suggestion
As highlighted in a [recent blog post](../10-14-a-tale-of-two-tools/index.md), going through cloud infrastructure with CLI can be tiresome and requires a lot of contextual knowledge because the API can differ between services. This is hardly conducive to effectively keeping tags under control. In the blog post, we also outlined how Resoto simplifies interaction with your cloud(s).
```